src/main/java/mate/academy/bookshop/controller/BookController.java

```diff
@@ -32,4 +36,16 @@ public BookDto findBookById(@PathVariable Long id) {
     public BookDto save(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        bookService.deleteById(id);
+    }
+
+    @PutMapping(value = "/{id}", consumes = {"application/json"})
```

Comment:
Specifying `consumes` attribute is not necessary here as it is the default value when you receive some class as a body.